common.sh

```diff
@@ -384,6 +384,12 @@ export AGENT_E2E_TEST_BOOT_MODE=${AGENT_E2E_TEST_BOOT_MODE:-"ISO"}
 # to open a firewall port
 export AGENT_PXE_SERVER_PORT=${AGENT_PXE_SERVER_PORT:-8089}
 
+# Ports that need to be open for platform "none". Hosts inside the libvirt firewalld zone
+# need to communicate with haproxy deployed on the libvirt host.
+export AGENT_MCS_PORT=22623
+export AGENT_KUBE_API_PORT=6443
+export AGENT_INGRESS_ROUTER_PORT=443
```

Comment:
What's the reason to open ports 443/6443 (at least 6443 shouldn't be already accessible)?

Reply:
6443 is not already accessible. If the port is not open, and after the 2 control plane nodes reboot after fetching their ignition, they will get stuck, do nothing, and block the installation from progressing. The reason is they cannot reach the kube-api running on the bootstrap node unless port 6443 is opened. The kube-api requests has to go through the external load balancer which lives outside libvirt firewalld zone.

If port 443 is not opened, one of the problems is it blocks the authentication operator.
```
[rwsu@nesting-vm dev-scripts]$ oc get co
NAME                                       VERSION       AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
authentication                             4.14.0-ec.3   False       True          False      7m37s   OAuthServerRouteEndpointAccessibleControllerAvailable: Get "https://oauth-openshift.apps.ostest.test.metalkube.org/healthz": dial tcp 192.168.111.1:443: connect: connection refused...
console                                    4.14.0-ec.3   False       True          False      19m     DeploymentAvailable: 0 replicas available for console deployme
nt...
ingress                                    4.14.0-ec.3   True        False         True       11m     The "default" ingress controller reports Degraded=True: Degrad
edConditions: One or more other status conditions indicate a degraded state: CanaryChecksSucceeding=False (CanaryChecksRepetitiveFailures: Canary route checks for t
he default ingress controller are failing)
```